packages/rag/retriever.py

```
@@ -0,0 +1,252 @@
+
+
+def get_query_embedding(query: str) -> list[float]:
+    """Clova Embedding v2 API로 쿼리를 임베딩 벡터로 변환"""
+    url = "https://clovastudio.stream.ntruss.com/v1/api-tools/embedding/v2/"
+    headers = {
+        "Authorization": f"Bearer {config.CLOVASTUDIO_API_KEY}",
+        "Content-Type": "application/json",
+    }
+    data = {"text": query}
+
+    response = requests.post(url, headers=headers, json=data)
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)